hapi/internal-protobufs/network/block_node_connections.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package com.hedera.hapi.network;
+
+option java_package = "com.hedera.node.internal.network.legacy";
+// <<<pbj.java_package = "com.hedera.node.internal.network">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+// Configuration for a single block node connection
+message BlockNodeConfig {
+    int32 priority = 1;
+    string address = 2;
+    int32 port = 3;
+}
+
+// Overall connection configuration for block nodes
+message BlockNodeConnectionInfo {
```

Comment:
Similar comment to the one about the `priority`: we can leave any additional configuration options out of the initial implementation and add them later e.g. `node_reselection_interval` & `max_simultaneous_connections`.